source/versioning.rst

```diff
@@ -3,15 +3,16 @@
 3-D Secure version determination
 ================================
 
-There are currently three 3-D Secure versions, ``1.0.2``, ``2.1.0`` and ``2.2.0``.
+There are currently three 3-D Secure versions, ``2.1.0``, ``2.2.0`` and ``2.3.1.1``.
 
 1. If the :ref:`preauth call <preauth-usage>` returns a :ref:`not_enrolled`, continue with 3DSv1.
-2. If ``acsInfoInd`` (`2.1.0
-   <specification_210.html#attr-cardRangeData-acsInfoInd>`_, `2.2.0
-   <specification_220.html#attr-cardRangeData-acsInfoInd>`_) is included, this
-   indicates if ACS supports challenges or only attempts.
+2. If ``acsInfoInd``
+   (`2.1.0 <specification_210.html#attr-CRD-acsInfoInd>`_,
+   `2.2.0 <specification_220.html#attr-CRD-acsInfoInd>`_,
+   `2.3.1.1 <specification_2311.html#attr-CRD-acsInfoInd>`_)
+   is included, this indicates if ACS supports challenges or only attempts.
 
-   To select between ``2.1.0`` and ``2.2.0``, we suggest using the highest version
+   To select between ``2.1.0``, ``2.2.0`` and ``2.3.1.1``, we suggest using the highest version
    that all parties support, including your implementation. Exemplified:
 
    - If ``dsEndProtocolVersion: 2.2.0`` and ``acsEndProtocolVersion: 2.1.0``, then the ACS
```

Comment:
We also need to reconsider this part a bit.
Depends on how the preauth response will be.
I believe @kbadk and I will have a talk about the preauth response.

Reply:
Do we really want to specify `2.3.1.1`? Wouldn't `2.3.1` be sufficient?